R/jackknifeafterboot.R

```diff
@@ -5,26 +5,26 @@ jab <- function(t, t0, starts, m=1, fn=sd) {
   
   jack.boot <- function(indices, xstar, f) {
```

Comment:
here I have to check whether or not this function really does what it should, but this is not related to the current pull request